src/components/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import styles from './TextArea.st.css';
+import {
+  TEXT_AREA_SUCCESS_ICON,
+  TEXT_AREA_DATA_HOOK,
+  TEXT_AREA_ERROR_ICON,
+  TEXT_AREA_ERROR_DESCRIPTION,
+} from './dataHooks';
+// tslint:disable-next-line:no-duplicate-imports
+import { ChangeEvent } from 'react';
+import { Tooltip } from '../Tooltip';
```

Comment:
This tooltip isn't the same one the `Textarea` should use.
However, we currently don't have one that fits for this component.
You can leave it like this for now, and i'll check with the designers.